proto/fairyring/common/shared_types.proto

```diff
@@ -9,27 +9,31 @@ option go_package = "github.com/Fairblock/fairyring/x/common/types";
 // RequestAggrKeyshare defines a struct for the data payload
 message RequestAggrKeyshare {
   string creator = 1;
+  // id can either be a rwquest id or a proposal id
   oneof id {
     string proposal_id = 2;
     string request_id = 3;
   }
     google.protobuf.Duration estimated_delay = 4 [(gogoproto.stdduration) = true];
 }
 
+// RequestAggrKeyshareResponse defines the response to the RequestAggrKeyshare message
 message RequestAggrKeyshareResponse {
```

Comment:
Let's make the following changes:

`AggrKeyshare` to `DecryptionKey`
so `RequestAggrKeyshare` becomes `RequestDecryptionKey`, etc...